roles/contiv_network/tasks/main.yml

```diff
@@ -20,21 +20,35 @@
 - name: download netmaster and netplugin
   get_url:
     validate_certs: "{{ validate_certs }}"
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-12-11-2015.20-54-40.UTC/netplugin-v0.0.0-12-11-2015.20-54-40.UTC.tar.bz2
+    url: https://github.com/contiv/netplugin/releases/download/v0.1-01-28-2016.03-55-05.UTC/netplugin-v0.1-01-28-2016.03-55-05.UTC.tar.bz2
```

Comment:
@vijayvikrant: The current installer script will pull in the latest changes from ansible and deploy services. If it breaks, there will be extra effort to check the error, if someone hits it meantime. The change is pretty simple. Change this line in "roles/contiv_network/templates/netplugin.j2" from NETPLUGIN_ARGS='-docker-plugin -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}' to NETPLUGIN_ARGS='-vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'